db/schema.rb

```diff
@@ -10,45 +10,55 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160915183926) do
+ActiveRecord::Schema.define(version: 20180129204835) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "api_consumers", force: :cascade do |t|
-    t.string   "public_api_key"
-    t.string   "private_api_key"
-    t.string   "contact_email"
-    t.datetime "created_at",      null: false
-    t.datetime "updated_at",      null: false
+  create_table "api_consumers", id: :serial, force: :cascade do |t|
+    t.string "public_api_key"
+    t.string "private_api_key"
+    t.string "contact_email"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
   end
 
-  create_table "measurements", force: :cascade do |t|
-    t.decimal  "temperature"
-    t.json     "custom_attributes"
-    t.integer  "sensor_id"
-    t.datetime "created_at",        null: false
-    t.datetime "updated_at",        null: false
-    t.index ["sensor_id"], name: "index_measurements_on_sensor_id", using: :btree
+  create_table "measurements", id: :serial, force: :cascade do |t|
+    t.decimal "temperature"
+    t.json "custom_attributes"
+    t.integer "sensor_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["sensor_id"], name: "index_measurements_on_sensor_id"
   end
 
-  create_table "sensors", force: :cascade do |t|
-    t.string   "device_name"
-    t.string   "caption"
-    t.integer  "sponsor_id"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
-    t.float    "latitude"
-    t.float    "longitude"
-    t.index ["sponsor_id"], name: "index_sensors_on_sponsor_id", using: :btree
+  create_table "sensors", id: :serial, force: :cascade do |t|
+    t.string "device_name"
+    t.string "caption"
+    t.integer "sponsor_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.float "latitude"
+    t.float "longitude"
+    t.index ["sponsor_id"], name: "index_sensors_on_sponsor_id"
   end
 
-  create_table "sponsors", force: :cascade do |t|
-    t.string   "name"
-    t.text     "description"
-    t.boolean  "active"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+  create_table "sponsor_images", force: :cascade do |t|
+    t.bigint "sponsor_id"
```

Comment:
Why is this a bigint, and `sponsor_id` in the `sensors` table is an integer?

Reply:
Rails migrated their datatypes somewhen. Shouldn't matter because `bigint` is bigger.